apps/daimo-mobile/src/model/account.ts

```diff
@@ -70,6 +72,24 @@ interface AccountV3 extends StoredModel {
   pushToken: string | null;
 }
 
+interface AccountV4 extends StoredModel {
```

Comment:
we might not need an `Account` migration

`isDeployed` should always be true. we send the first transaction at account creation time, claiming the name

so by the time we call `setAccount` to save to storage, it's already deployed

Reply:
> we send the first transaction at account creation time, claiming the name

I don't think so? Name claim just assigns the name to the counter factual address, but doesn't deploy it atm I believe?

Reply:
Example, https://goerli.basescan.org/tx/0x13d5ea27049dc4f84d94b3ba0616658675a2e75de4b212c93d0a308f9712c6db is deployed at first transfer.

Reply:
you're right. sending the tx from the faucet address was a shortcut, i think i have a TODO in there...

given we're spotting gas for the name claim, we can send it from the account (currently paying gas via prefund, another shortcut, soon via paymaster) which will instantiate it

if we don't do that, the first outbound transfer will have higher than normal fees

simplifying the stored account is a bonus

Reply:
> given we're spotting gas for the name claim, we can send it from the account which will instantiate it

hm, looking more closely at this right now and i think we should still send the tx from our faucet/api backend address.

Benefits:
- Saves a step in user onboarding flow (biometrics for deploy signature).
- If the server deploys and returns the deployed address before we initialize the userop package, it doesn't need to care about calling `initCode`/`getSenderAddress` and we can get rid of the RPC call there. Server is trusted but that's fine I think.

Reply:
Marking resolved: done in #160 and rebased this on top of that.